misc/docker/arch/Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM archlinux/base:latest
```

Comment:
archlinux/base [does not exist](https://hub.docker.com/r/archlinux/base/), so I changed as follow